b2sdk/account_info/sqlite_account_info.py

```diff
@@ -91,10 +93,25 @@ def __init__(self, file_name=None, last_upgrade_to_run=None, profile: str | None
         )
 
     @classmethod
-    def _get_user_account_info_path(cls, file_name=None, profile=None):
+    def get_xdg_config_path(cls) -> str | None:
+        """
+        Return XDG config path if the OS is XDG-compatible (Linux, BSD), None otherwise.
+
+        If $XDG_CONFIG_HOME is empty but the OS is XDG compliant, fallback to ~/.config as expected by XDG standard.
+        """
+        xdg_config_home = os.getenv(XDG_CONFIG_HOME_ENV_VAR)
+        if xdg_config_home or sys.platform.startswith(('linux', 'freebsd', 'openbsd', 'netbsd')):
```

Comment:
ideally we would use `platformdirs` package, but since I guess it doesn't quite make sense to add another 3rd party dependency just for hits, maybe lets take a lesson from them and make this into
```suggestion
        if xdg_config_home or not sys.platform == "win32":
```
https://github.com/platformdirs/platformdirs/blob/main/src/platformdirs/__init__.py#L24

Only suggesting that, since it seems it will be very easy to miss something, and `platformdirs` ppl are the one most likely to have get it right since a lot of packages rely on them https://github.com/platformdirs/platformdirs/network/dependents?dependent_type=PACKAGE

Reply:
their decision logic seems to be `sys.platform not in ('win32', 'linux')`, we can use that without adding the whole package as a dependency, that feels like an overkill.